jenkins/spark-nightly-build.sh

```diff
@@ -20,6 +20,10 @@ set -ex
 . jenkins/version-def.sh
 
 mvn -U -B clean deploy $MVN_URM_MIRROR -Dmaven.repo.local=$WORKSPACE/.m2
+# Run unit tests against other spark versions
+mvn -U -B -Pspark301tests test $MVN_URM_MIRROR -Dmaven.repo.local=$WORKSPACE/.m2
```

Comment:
Same comment here we probably want to disable the python tests unless/until we download a python 3.0.1 release and can set SPARK_HOME to point to it.

Reply:
SPARK HOME isn't set in nightly so the only ones run are tpch and other scala, which I was intending to still run.

Reply:
note we have separate integration builds for those that I'm testing now to make sure 3.0.1 passes now